srcpkgs/metis/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'metis'
+pkgname=metis
+version=5.1.0
+revision=1
+build_style=gnu-makefile
+hostmakedepends="cmake"
```

Comment:
why not just use the cmake build style?

Reply:
While still overriding `do_configure()`? If that's okay, then sure.

Reply:
you shouldn't need to touch their weird makefile -- it runs cmake internally

Reply:
The makefile does do one thing - resolve the absolute path the the `GKlib` folder.
Could have been done within CMake, but doing it ourselves isn't too bad.